lib/graphql-ast.explorer.ts

```diff
@@ -46,6 +46,25 @@ export interface DefinitionsGeneratorOptions {
    * @default false
    */
   skipResolverArgs?: boolean;
+
+  /**
+   * If provided, specifies a default generated TypeScript type for custom scalars.
+   * @default 'any'
+   */
+  defaultScalarType?: string;
+
+  /**
+   * If provided, specifies a mapping of types to use for custom scalars
+   * @default undefined
+   */
+  customScalarTypeMapping?: Record<string, string>;
```

Comment:
@davidgoli One last minor improvement (should be easy to add) - what do you think about supporting the `Record<string, string | Function>` type instead? Example usage:

```typescript
customScalarTypeMapping: {
  DateTime: Date,
},
```
Internally, we would simply check if a value is === 'function' and if so, read the `.name` property (e.g., `Date.name`). The rest would remain the same

Reply:
Done, and pushed!